sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/GalleriesOperations.cs

```diff
@@ -134,7 +138,7 @@ public async Task<AzureOperationResponse<Gallery>> UpdateWithHttpMessagesAsync(s
         /// <return>
         /// A response object containing the response body and response headers.
         /// </return>
-        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<AzureOperationResponse<Gallery>> GetWithHttpMessagesAsync(string resourceGroupName, string galleryName, string select = default(string), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
This is a breaking change in dotnet with the introduction of the new parameter select. Could the parameter be moved to the end?

Reply:
This was removed after the latest changes were added to compute-2020-12